Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryResult.php

```diff
@@ -214,6 +214,23 @@ public function searchHitForNode(NodeInterface $node)
     }
 
     /**
+     * Returns the array with all sort values for a given node. The values are fetched from the raw content
+     * ElasticSearch returns within the hit data
+     *
+     * @param NodeInterface $node
+     * @return array
+     */
+    public function getSearchValuesForNode(NodeInterface $node)
```

Comment:
The behavior of this method does not fit to its name ...
Do we really need this method at all?